docs/quantum_chess/concepts.ipynb

```diff
@@ -0,0 +1,2063 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "-SXDOmR2MXAs"
+   },
+   "source": [
+    "##### Copyright 2020 The Cirq Developers"
```

Comment:
The other notebooks in ReCirq have Copyright 2020 Google. I make no claims as to which is the correct one, but if I had to guess: Cirq uses "The Cirq Developers" because they have an AUTHORS file (which just lists Google) but ReCirq does not